README.md

```diff
@@ -33,6 +33,88 @@ updating the typesystems as well.
 
 # Building
 
+## General
+
+Building PythonQt requires a couple of steps.
+Follow these instructions in order to get a correctly built PythonQt runtime and Qt bindings.
+
+### ⚠️ Pregenerated Bindings Warning ⚠️
+
+It is highly recommended to build the Qt bindings yourself and not(!) use the pregenerated ones. To ensure that, delete the pregenerated directory starting with `generated_cpp_*`.
```

Comment:
Maybe add some reasoning, like "to ensure that the bindings are always compatible with your Qt version".

Reply:
From experience, 5.6 pre-gen bindings worked fine with 5.9, 5.12, and 5.15 albeit you never get the newer constructs and methods of course. 5.15 pre-gen bindings work fine with 5.15.

Reply:
I used it with 5.11 and I got issues see #230

Reply:
@mrbean-bremen I added your mentioned reasion